src/cpp/editor/overlay.cpp

```diff
@@ -24043,7 +24043,7 @@ overlay gOverlayDatabase[NUM_OVERLAYS] =
     978,
     919,
     0,
-    37,
+    35,
```

Comment:
Typos in this data structure are scary.

Reply:
In fact it wasn't a typo at first, I just forgot that I made a new overlay for town shadow too (which 37 is)